lib/markus_configurator.rb

```diff
@@ -51,7 +51,7 @@ def markus_config_repository_type
     if defined? REPOSITORY_TYPE
       return REPOSITORY_TYPE
     else
-      return 'git'
+      return "git"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.